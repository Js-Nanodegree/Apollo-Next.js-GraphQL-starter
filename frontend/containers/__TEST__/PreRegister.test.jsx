import renderer from 'react-test-renderer';
import PreRegisterContainer from '../PreRegister';

describe('PreRegister container', () => {
  it('should match snapshot', () => {
    const handleChange = jest.fn();
    const invite = jest.fn();

    const tree = renderer
      .create(
        <PreRegisterContainer
          handleChange={handleChange}
          invite={invite}
          email=''
          loading={false}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
