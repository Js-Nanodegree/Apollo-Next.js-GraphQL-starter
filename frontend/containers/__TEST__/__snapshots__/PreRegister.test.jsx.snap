// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreRegister container should match snapshot 1`] = `
<div
  className="qepopj-0 gBRnsH"
>
  <form
    className="cw1g19-0 cGYWuk"
    onSubmit={[Function]}
  >
    <h1>
      Register
    </h1>
    <p>
      Enter your email address and click submit
    </p>
    <div
      className="MuiFormControl-root-1 MuiFormControl-fullWidth-4"
    >
      <label
        className="MuiFormLabel-root-16 MuiFormLabel-required-21 MuiInputLabel-required-9 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
        data-shrink={false}
      >
        Email address
        <span
          className="MuiFormLabel-asterisk-22"
        >
          â€‰*
        </span>
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-fullWidth-45 MuiInput-fullWidth-30 MuiInputBase-formControl-37 MuiInput-formControl-24"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiInput-input-31"
          disabled={false}
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
    </div>
    <br />
    <br />
    <button
      className="MuiButtonBase-root-79 MuiButtonBase-disabled-80 MuiButton-root-53 MuiButton-contained-64 MuiButton-containedPrimary-65 MuiButton-raised-67 MuiButton-raisedPrimary-68 MuiButton-disabled-73 MuiButton-fullWidth-78"
      disabled={true}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="-1"
      type="submit"
    >
      <span
        className="MuiButton-label-54"
      >
        SUBMIT
      </span>
    </button>
    <p>
      <a
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </p>
  </form>
</div>
`;
